app/lib/capital_subtotals.rb

```diff
@@ -1,10 +1,17 @@
 class CapitalSubtotals
   class << self
-    def blank
-      new(applicant_capital_subtotals: PersonCapitalSubtotals.blank,
-          partner_capital_subtotals: PersonCapitalSubtotals.blank,
-          capital_contribution: 0,
-          combined_assessed_capital: 0)
+    def unassessed(applicant:, partner:, submission_date:)
+      new(
+        applicant_capital_subtotals: PersonCapitalSubtotals.unassessed(vehicles: assessed_vehicles(applicant, submission_date)),
+        partner_capital_subtotals: PersonCapitalSubtotals.unassessed(vehicles: assessed_vehicles(partner, submission_date)),
+        capital_contribution: 0,
+        combined_assessed_capital: 0,
+      )
+    end
+
+    def assessed_vehicles(person, submission_date)
+      vehicles = person&.vehicles || []
+      Assessors::VehicleAssessor.call(vehicles, submission_date)
     end
```

Comment:
This is incorrect - we need to do what we were doing in the previous PR (producing a dummy assessment)